src/webapp/views.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import permissions
+from rest_framework.schemas import get_schema_view
+from rest_framework_json_api.schemas.openapi import SchemaGenerator
+
+schema_view = get_schema_view(
+    title="<NAME THE SCHEMA>",
+    description="e.g. <SCHEMA OF MY PROJECT>",
```

Comment:
@km1230 Similarly, to the line above, we could set this to `f"Schema for {settings.PROJECT_NAME}"`